api/app/scripts/reset_db_fixtures.sh

```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env bash
 set -e
 
-# We need below to create the params file on container start
+environment=${1:-development}
+
 confd -onetime -backend env
 php app/console doctrine:fixtures:load --no-interaction
+
+# Only run the test fixtures load if the environment is 'local'
+if [ "$environment" == "local" ]; then
+  php app/console doctrine:fixtures:load --no-interaction --env=test
```

Comment:
These fixtures are used by users on the training environment. Will this affect those users?

Reply:
Nah. Training environment was just having the usual: php app/console doctrine:fixtures:load --no-interaction previously and this will do the same.

The reason we do additional reset with different flag in local is because the local tests use a different database as defined in the .yml files so we have to reset both.